packages/status-badge/src/consts/iconMap.ts

```diff
@@ -17,66 +12,165 @@ import { StopBrickCompactMIcon } from '@alfalab/icons-glyph/StopBrickCompactMIco
 import { StopBrickMIcon } from '@alfalab/icons-glyph/StopBrickMIcon';
 import { StopBrickSIcon } from '@alfalab/icons-glyph/StopBrickSIcon';
 
+import type {
+    StatusBadgeCustomIcon,
+    StatusBadgeIcon,
+    StatusBadgeSizes,
+    StatusBadgeViews,
+} from '..';
+import { combineIcons } from '../hooks/useStatusBadgeIcon';
+
+import { ICON_MAP_BLANK } from './iconMap.blank';
+
 export type IconMap = Record<
     StatusBadgeViews,
     Record<NonNullable<StatusBadgeSizes>, StatusBadgeIcon>
 >;
 
-export const ICON_MAP: IconMap = {
+const ICON_MAP_16: StatusBadgeCustomIcon = {
     'positive-checkmark': {
```

Comment:
Если единая мапка делится на несколько разных, которые содержат перечисления единственного размера то теряется смысл в цифре `16:` , `20:` и т. д.  
```
16: CheckmarkCompactSIcon
```

```suggestion
    'positive-checkmark': CheckmarkCompactSIcon
```